modules/abstract-utxo/src/parseOutput.ts

```diff
@@ -228,6 +230,22 @@ export async function parseOutput({
      * details are fetched on the wallet, a local address validation is run, whose errors however are generated
      * client-side and can therefore be analyzed with more granularity and type checking.
      */
+
+    /**
+     * In order to minimize API requests, we assume that explicit recipients are always external when the
+     * recipient list is > 1000 This is not always a valid assumption and can lead to unexpected behavior.
+     */
+    assert(txParams.recipients);
+    if (txParams.recipients?.length > RECIPIENT_THRESHOLD) {
+      const isCurrentAddressInRecipients = txParams.recipients?.some((recipient) =>
```

Comment:
Shouldn't need the `?`s with the assert
```suggestion
    if (txParams.recipients.length > RECIPIENT_THRESHOLD) {
      const isCurrentAddressInRecipients = txParams.recipients.some((recipient) =>
```